README.md

```diff
@@ -28,6 +29,22 @@ sudo dpkg -i omnect-cli_<version>_amd64.deb
 ```
 **Note**: `dpkg` lists necessary runtime dependencies in case they are not present.
 
+## Docker image
+
+`omnect-cli` is also provided as docker image.<br>
+
+Example usage:
+
+```ssh
+docker run --rm -it \
+  --mount type=bind,source="$(pwd)"/,target=/source \
+  --env RUST_LOG=info \
```

Comment:
is that not the default?

Reply:
since we don't have info logs I changed to debug